cmd/allocator/main.go

```diff
@@ -212,21 +212,21 @@ func newServiceHandler(kubeClient kubernetes.Interface, agonesClient versioned.I
 }
 
 // Set up our client which we will use to call the API
-func getClients() (*kubernetes.Clientset, *versioned.Clientset, error) {
+func getClients() (kubeClient *kubernetes.Clientset, agonesClient *versioned.Clientset, err error) {
```

Comment:
I don't understand the suggested changes to this function; the old way looks more correct to me stylistically. Do you know which check triggered this?

Reply:
Yeah, I'm not a fan of named return variables either.

Reply:
I think the idea behind this check was to make function signature more readable from the first line, especially when it returns same 3 types for instance, which can be misleading without names.
But as we have 3 votes to not use them I will remove this check today.